packages/web-runtime/src/components/Topbar/UserMenu.vue

```diff
@@ -136,25 +136,18 @@ export default defineComponent({
   },
   setup() {
     const route = useRoute()
+    const themeStore = useThemeStore()
+
     const loginLink = computed(() => {
       return {
         name: 'login',
         query: { redirectUrl: unref(route).fullPath }
       }
     })
-    const store = useStore()
-    const imprintUrl = computed(() => {
-      return (
-        store.getters.configuration.currentTheme.general?.imprintUrl ||
-        store.getters.configuration.options.imprintUrl
-      )
-    })
-    const privacyUrl = computed(() => {
-      return (
-        store.getters.configuration.currentTheme.general?.privacyUrl ||
-        store.getters.configuration.options.privacyUrl
-      )
-    })
+
+    const imprintUrl = computed(() => themeStore.currentTheme.common.urls.imprint)
```

Comment:
For the record, we might need to re-introduce setting the imprint and privacy urls via config.json for short term deployment reasons.